js/packages/web/src/views/auctionCreate/index.tsx

```diff
@@ -540,10 +544,12 @@ export const AuctionCreateView = () => {
   const categoryStep = (
     <CategoryStep
       confirm={(category: AuctionCategory) => {
-        setAttributes({
-          ...attributes,
-          category,
-        });
+        if (category !== attributes.category) {
+          setAttributes({
+            ...freshAttributes,
+            category,
+          });
+        }
```

Comment:
Instead of reset on confirm can we clear out selects in the same on change as picking a category?

Reply:
Yep just would have to pass the setAttributes state setter into categoryStep -- I will change it to that way instead.  Thanks Kyle